cucumber-jparest/src/test/java/uk/gov/homeoffice/digital/sas/cucumberjparest/utils/SecureRandomStringGeneratorTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest.utils;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class SecureRandomStringGeneratorTest {
+
+  private static Stream<Arguments> stringToMapTestData() {
+    return Stream.of(
+        Arguments.of(10),
+        Arguments.of(5),
+        Arguments.of(0)
+    );
+  }
+
+  @ParameterizedTest
+  @MethodSource("stringToMapTestData")
+  void GIVEN_string_size_WHEN_generating_random_string_THEN_produces_the_expected_string(
```

Comment:
the test naming convention we have been sticking for the jpaRest unit tests is:
`[method]_[constraint]_[expectation]` which could also be represented as `[method]_[given]_[then]`
In this example it might look something like:
`randomAlphabetic_stringSizeProvided_randomStringEqualsStringSize`

The same could be applied to the other few tests written in this format (if taken onboard)

Reply:
Actioned